controllers/v1/reportsController.js

```diff
@@ -29,7 +29,44 @@ module.exports = class Reports {
   static get name() {
     return 'submissions';
   }
-
+  
+ async fetch  (req, res) {
+
+  try {
```

Comment:
@praveenKDass We have to finalise @borkarsaish65 changes and incorporate them correctly here after testing